src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -205,105 +202,171 @@ int shardId() {
         return this.shardId;
     }
 
-    public String script() {
+    public Script scriptObject() {
```

Comment:
I think it's fine this way but it's important to not forget to rename in master afterwards.